checker/jdk/nullness/src/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -1755,7 +1758,9 @@ else if ((val = mappingFunction.apply(key)) != null) {
      * @throws RuntimeException or Error if the remappingFunction does so,
      *         in which case the mapping is unchanged
      */
-    public V computeIfPresent(K key, BiFunction<? super K, ? super V, ? extends V> remappingFunction) {
+    public @PolyNull V computeIfPresent(
+            K key,
+            BiFunction<? super K, ? super V, ? extends @PolyNull V> remappingFunction) {
```

Comment:
Missing @Nullable here.

Reply:
Can you explain this comment?  Does this mean you need to update your pull request?

Reply:
Please check the pull request (on GitHub).  If it needs changes, please make them.  If it's correct, please verify that.  Thanks!

PS: It's generally better to answer on the GitHub website.  Even though it's convenient to make a reply by email, that tends to put the response in the wrong location.

Welcome to GitHub; you'll be using it like a pro in no time.

Reply:
@awturner 
https://github.com/typetools/checker-framework/pull/2078/commits/e83e00de17a07e62a9623333a5382bda8d850b7f is part of the pull request, but it added `@Nullable` on line 1855 below, not here.
Did you also intend to have an annotation here?

Reply:
@awturner Werner answered your question; could you take action or explain https://github.com/typetools/checker-framework/pull/2078/files#r204313117 ?
Thanks!

Reply:
I have placed the @Nullable where I intended to.

Reply:
So what did you mean with the `Missing @nullable here.` comment that started this thread?
Why the different signatures for the two methods?

Reply:
I simply put the comment in the wrong place. It was intended as a note to self that I needed to fix it.

They are different signatures because the methods do different things: ConcurrentHashMap does not allow null values, so the current value passed to computeIfPresent is not null; the current value passed to compute is null if there is no mapping present.